Source/Scene/TileMapServiceImageryProvider.js

```diff
@@ -97,30 +99,14 @@ define([
         }
         //>>includeEnd('debug');
 
-        var url = options.url;
-
+        var url = appendForwardSlash(options.url);
         this._url = url;
-        this._ready = false;
```

Comment:
Ah nevermind, we are going to revert all changes to this file and just deprecate it, right?